views/pages/partials/museums.ejs

```diff
@@ -0,0 +1,7 @@
+<div class = "artworkDiv">
+    <p class="museum"><%= artwork.museum%></p>
+    <p class="artworkDescription"><%= artwork.cities%></p>
```

Comment:
This partial is not displaying the correct things. It should just show a list of museums and total artwork count for each.

Reply:
I deleted this partial because this was going to be for the feature I wasn't able to finish.